apollo-federation/src/schema/field_set.rs

```diff
@@ -57,15 +57,25 @@ pub(crate) fn parse_field_set(
     schema: &ValidFederationSchema,
     parent_type_name: NamedType,
     field_set: &str,
+    validate: bool,
```

Comment:
How about `parse_field_set` and `parse_and_validate_field_set`?
I find bool arguments like `true` are not so reader friendly. But, it's not a big deal.

Reply:
I think most IDEs these days have some form of inlay parameter names, e.g. for vscode
<img width="437" alt="Screenshot 2025-04-19 at 5 56 43 PM" src="https://github.com/user-attachments/assets/551bce67-e91c-492e-bd09-7948a92e58b4" />
so I haven't worried too much about it historically, but if anyone has a strong preference I'm fine with changing it.